drivers/sensor/isl29035/isl29035_trigger.c

```diff
@@ -154,6 +153,8 @@ int isl29035_trigger_set(const struct device *dev,
 
 	/* enable interrupt callback */
 	setup_int(dev, true);
+	if (gpio_pin_get_dt(&config->int_gpio) > 0) {
+		handle_int(dev);
```

Comment:
This changes the driver behavior. Is this change intentional, and if so, can you explain the reason in the commit message?

Reply:
This was not intentional, thank you! It's fixed now.